lib/certman/resource/acm.rb

```diff
@@ -35,7 +35,7 @@ def certificate_exist?
         current_cert = acm.list_certificates.certificate_summary_list.find do |cert|
           cert.domain_name == @domain
         end
-        current_cert
+        @cert_arn = current_cert.certificate_arn
```

Comment:
@MetricMike

`current_cert` may be `nil`. So could you fix it ?

Example:

```ruby
@cert_arn = current_cert.certificate_arn if current_cert
```

Reply:
9c579f4 fixes this with the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/), but that's only available in ruby 2.3 so I bumped the version. Is that okay?